lms/djangoapps/courseware/tests/test_module_render.py

```diff
@@ -1103,7 +1103,7 @@ def test_static_badlink_rewrite(self):
         result_fragment = module.render(STUDENT_VIEW)
 
         self.assertIn(
-            '/c4x/{org}/{course}/asset/_file.jpg'.format(
+            '/c4x/{org}/{course}/asset/file.jpg'.format(
```

Comment:
Right on.  Yeah, just for my sanity, I checked that particular PDF asset, and it is in fact stored with the forward slashes as underscores: `asset-v1:MITx+6.004.2x+3T2015+type@asset+block@pdfs_isa_c9s1v1.pdf`.  

And then just to verify what happens when we try to handle this path against this course:

    >>> mit_course = CourseLocator(org='MITx', course='6.004.2x', run='3T2015')
    >>> StaticContent.get_canonicalized_asset_path(mit_course, u'/static/pdfs/isa/c9s1v1.pdf', u'')
    u'/asset-v1:MITx+6.004.2x+3T2015+type@asset+block@pdfs_isa_c9s1v1.pdf'

So yeah, this should not be an issue. :)